content/blog/pinpoint-kubernetes/index.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "How Pinpoint Manages Kubernetes with Pulumi"
+date: 2020-08-14
+meta_desc: "Pinpoint uses Pulumi to streamline deployments and scale Kubernetes"
+meta_image: meta.png
+authors:
+    - sophia-parafina
```

Comment:
Can we add the Pinpoint team names here?

Reply:
yes, I'll add them